mscore/preferences.cpp

```diff
@@ -98,6 +98,8 @@ void Preferences::init(bool storeInMemoryOnly)
             {PREF_APP_STARTUP_STARTSCORE,                          new StringPreference(":/data/My_First_Score.mscx", false)},
             {PREF_UI_APP_STARTUP_SHOWTOURS,                        new BoolPreference(true)},
             {PREF_APP_WORKSPACE,                                   new StringPreference("Basic", false)},
+            {PREF_APP_TELEMETRY_ALLOWED,                           new BoolPreference(false)},
+            {PREF_APP_STARTUP_TELEMETRY_ACCESS_REQUESTED,          new StringPreference("", false)},
```

Comment:
IMHO this one here should be visible in the Advanced settings, like it needed for testing